packages/fx-core/templates/plugins/resource/bot/bicep/param.template.bicep

```diff
@@ -8,4 +8,4 @@ param bot_webAppSKU string = 'F1'
 param bot_serviceSKU string = 'F1'
 param bot_sitesName string = '${resourceBaseName}-bot-sites'
 param bot_displayName string = '${resourceBaseName}-bot-displayname'
-param authLoginUriSuffix string = 'auth-start.html'
+param authLoginUriSuffix string = 'public'
```

Comment:
why we change this value? based on existing bicep file the login url is `var initiateLoginEndpoint = uri(botEndpoint, authLoginUriSuffix)`, seems not the expected one after changed the value?

Reply:
The value is supposed to be 'public' instead of 'auth-start.html' here so fix the bot bicep error here.
See the non-arm-support implementation [here](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/plugins/resource/bot/plugin.ts#L349).
The expected redirect value is something like `"redirectUri": "https://tabandbotbt47702d.azurewebsites.net/public"`.

Reply:
but it misses the auth-start.html part... can we see what's the magics behind this?

Reply:
Fixed.